benchmark/async_hooks/current-resource-vs-destroy.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
```

Comment:
IMO some comments explaining what this benchmark is testing will help readers going forward.